incubator/group/README.md

```diff
@@ -0,0 +1,64 @@
+# Group Module
+
+## Group
+
+A group is simply an aggregation of accounts with associated weights. It is not
+an account and doesn't have a balance. It doesn't in and of itself have any
+sort of voting or decision power. It does have an "administrator" which has
+the power to add, remove and update members in the group. Note that a
+group account could be an administrator of a group.
+
+## Group Account
+
+A group account is an account associated with a group and a decision policy.
+Group accounts are abstracted from groups because a single group may have
+multiple decision policies for different types of actions. Managing group
+membership separately from decision policies results in the least overhead
+and keeps membership consistent across different policies. The pattern that
+is recommended is to have a single master group account for a given group,
+and then to create separate group accounts with different decision policies
+and delegate the desired permissions to from the master account to
+those "sub-accounts" using the `msg_authorization` module.
+
+
+## Decision Policy
+
+A decision policy is the mechanism by which members of a group can vote on 
+proposals.
+
+All decision policies generally would have a minimum and maximum voting window.
+The minimum voting window is the minimum amount of time that must pass in order
+for a proposal to potentially pass, and it may be set to 0. The maximum voting
+window is the maximum time that a proposal may be voted on before it is closed.
+Both of these values must be less than a chain-wide max voting window parameter.
+
+### Threshold decision policy
```

Comment:
This is the only decision policy to be implemented as part of the first work, right?

But the design should be expandable to other algorithms?
Or just different thresholds for different actions (add a member vs move some funds)?

Reply:
Yes, the idea is to support more policies like percent going forward, but just the minimal to start. I should actually change this to an interface in the spec.

Reply:
Switched to an interface, and actually updated this to use `google.protobuf.Any` for both `DecisionPolicy` and `sdk.Msg`. I think this is simpler than going with the `oneof` option for extensibility in spite of the overhead. What do you think?

Reply:
I personally dislike any and prefer explicit oneof.
Having a canonical list defined somewhere is a great tool.

That said, my preferences are often at odds with the general cosmos sdk aesthetic, so feel free to use any

Reply:
👍 For me `oneof` provides a good level of documentation as well. Clients may also generate code based on this. That should be "our" default

Reply:
For the cosmos sdk we are generally using `oneof`, but there are places where it's cumbersome. Will update this spec to use `oneof` for those cases.